packages/app/src/cli/models/extensions/specifications/app_config_webhook_schemas/webhooks_schema.ts

```diff
@@ -3,8 +3,13 @@ import {webhookValidator} from '../validation/app_config_webhook.js'
 import {WebhookSubscriptionUriValidation} from '../validation/common.js'
 import {SingleWebhookSubscriptionSchema} from '../app_config_webhook_subscription.js'
 import {mergeAllWebhooks} from '../transform/app_config_webhook.js'
+import {selectDeveloperPlatformClient} from '../../../../utilities/developer-platform-client.js'
+import {requestApiVersions} from '../../../../services/webhook/request-api-versions.js'
 import {zod} from '@shopify/cli-kit/node/schema'
 
+const developerPlatformClient = selectDeveloperPlatformClient()
+const acceptedApiVersions = await requestApiVersions(developerPlatformClient)
```

Comment:
Is there a way we can get all the accepted api versions without calling the client? I think this is breaking the unit tests